samples/subsys/fs/ext2/README.rst

```diff
@@ -0,0 +1,39 @@
+Ext2 Sample application
+#######################
+
+Overview
+********
+
+This sample app uses the Ext2 file system driver to mount an SDHC card connected over an SPI bus.
+
+Requirements
+************
+
+This sample requires SD card support and a microSD card formatted with Ext2 filesystem.
+Restrictions:
+
+* max file system size is:
+
+   .. code-block:: none
+
+      - 8MB for 1k blocks
+      - 32MB for 2k blocks
+      - 128MB for 4k blocks
+
+* file system has to be created with specific options (the implementation doesn't support all ext2 features)
+
+   .. code-block:: none
+
+      mkfs.ext2 /dev/<device> -b <block size> -I 128 -O ^ext_attr,^resize_inode,^dir_index,^sparse_super,^large_file
```

Comment:
Would add the final parameter here or at least details or it because people are likely going to be using cards that are 4GB or larger

Reply:
What do you mean by 'final parameter'? This is the command that works for me.

Also I will describe how to create partition with desired size as you mentioned in next comment.

Reply:
I was formatting on full card, so without a value it would be GBs in size, I had to use 32768 to get a small enough partition

Reply:
Okay. I was creating a partition with desired size and then formatted. I have described my approach in the README. Is it okay?